stripe/src/test/java/com/stripe/android/view/ShippingInfoWidgetTest.kt

```diff
@@ -273,5 +287,18 @@ class ShippingInfoWidgetTest {
             "Fake Name",
             "(123) 456 - 7890"
         )
+
+        private val SHIPPING_INFO_CA = ShippingInformation(
+                Address.Builder()
+                        .setCity("Ontario")
+                        .setState("Ontario")
+                        .setCountry("CA")
+                        .setLine1("185 Berry St")
+                        .setLine2("10th Floor")
+                        .setPostalCode("M4B1B5")
+                        .build(),
+                "Fake Name",
+                "416-759-0260"
```

Comment:
Let's line up the indentation with `SHIPPING_INFO`.

Reply:
Thanks, will fix the indention.